src/compiler/types.ts

```diff
@@ -2415,6 +2416,7 @@ namespace ts {
         jsx?: JsxEmit;
         reactNamespace?: string;
         listFiles?: boolean;
+        librarySearchPaths?: string[];

```

Comment:
i do not see parsing to this anywhere. has the changes to commandlineparser been missed intentionally?